### PR TITLE
Add permissions in the in-cluster RBAC doc

### PR DESCRIPTION
**Pull request by tnorris**
Clusterlint is rad, while trying to deploy it at work, I ran into API permission errors. Someone else might be trying to do this, so here's a docfix 😄 

This patch does three things:
- Add missing permissions for what seems like newer checks and API Groups
- Sorts the list of permissions
- Changes the RoleBinding to a ClusterRoleBinding so permissions can be picked up by the ServiceAccount

### CODE REVIEW

**Comment by tnorris** [comment]
Hi @sundowndev, sorry for the ping, but you're someone that touched the RBAC docs. Do you mind taking a look at this? 😃

File: docs/RBAC.md
Comment (sundowndev) [inline docs/RBAC.md:30]: I think we can remove `volumesnapshots` here since it's covered below

**Review by sundowndev — CHANGES_REQUESTED** [review]
Thanks for your PR @tnorris 🙏🏻 

I left few comments

File: docs/RBAC.md
Comment (sundowndev) [inline docs/RBAC.md:56]: Since it's now cluster-scoped, you need to remove the namespace here

**Review by sundowndev — APPROVED** [review]
LGTM - I'll let the core team review and merge now

cc @varshavaradarajan @adamwg

File: docs/RBAC.md
Comment (adamwg) [inline docs/RBAC.md:29]: Looks like an extra `]` at the end of this line.

**Review by adamwg — COMMENTED** [review]
One small yaml issue, otherwise this looks good to me.

**Review by varshavaradarajan — APPROVED** [review]

**Comment by varshavaradarajan** [comment]
Thank you @tnorris and @sundowndev!

**Pull request merged**
